### DISCUSSION TITLE
Possible to skip compression if archive already exists?

### DISCUSSION THREAD

**Discussion by gingerbeardman** [aonez/Keka#1174] [Q&A]
Currently if I compress using Keka and the archive already exists, a second archive called "filename 2.zip" is created.

But in this instance I'd like to skip the compression entirely if the archive already exists.

Is this possible?

**Comment by aonez** [comment]
There’s no such feature (unless you use the save dialog), but implementing this one shouldn’t be too hard. Anyway is this something you see wanting to work like this permanently?

**Comment by gingerbeardman** [comment]
It would be an option. 

Most of the time you would want the new archive. In this odd case I do not want the new archive as I know it will be the same as the existing one (one file per archive)

I ended up writing a shell script to do it for ~300 files.

Would have been nice to do it quickly with less hassle in Keka.

I will need to do it again in future, quarterly, but now I've written the script I will probably use that until it breaks.
